iree/tools/utils/image_util.c

```diff
@@ -0,0 +1,192 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include "iree/tools/utils/image_util.h"
+
+#include <math.h>
+
+#include "iree/base/internal/flags.h"
+#include "stb_image.h"
+
+IREE_FLAG(string, input_range, "0 1",
+          "Float input dynamic range separated by the space or comma. "
+          "Default to \"0 1\"");
```

Comment:
1. Can you elaborate on how this should be used? Could this value change between loaded images or uses of IREE within the same process? A flag might not be the best option here.
2. If only two values are accepted, how about two float flags (min/max) instead of a single string flag that comes with extra parsing?

Reply:
Split up the range into two flags. The values should be fixed per model, depending on how the model designer defines the dynamic range of the input image for the float model.
(see the discussion in https://github.com/tensorflow/hub/issues/739)

Reply:
If the values can change between models, they should not be flags. IREE supports loading and running multiple different models in the same process.

Reply:
Okay, I can input them in the function argument and set the default in the header.